### ISSUE TITLE
gzY creates error

### ISSUE THREAD

**Issue by Mikee-3000** [saulpw/visidata#193]
Hi Saul, 

I've updated to v1.4, installed via Pip in Anaconda on Linux.

When I select multiple rows and press gzY, it errors out with `SyntaxError: EOL while scanning string literal`.

CTRL+E gives this stack trace:
```
Traceback (most recent call last):
  File "/home/mike/anaconda3/lib/python3.6/site-packages/visidata/vdtui.py", line 1070, in exec_command
    exec(cmd.execstr, vdglobals, LazyMap(self))
  File "<string>", line 1
    copyToClipboard("
SyntaxError: EOL while scanning string literal
```
This bug is related to #186 (gzY sometimes doesn't copy selected rows). 

Let me know if you need any more information.

Thank you 

Mike

**Comment by saulpw** [comment]
Thanks for the bug report, @Mikee-3000, sorry about that.  That was a silly error on my part.  It will be fixed in 1.4.1, released very soon (maybe even this weekend).

**Comment by Mikee-3000** [comment]
Thank you very much!
